library/src/main/java/com/novoda/downloadmanager/Migrator.java

```diff
@@ -13,5 +15,10 @@
         public void migrate() {
             // no-op.
         }
+
+        @Override
+        public boolean isRunning() {
```

Comment:
What happens if you actually return the NO_OP? I think that is not processed anywhere

Reply:
It executes the `Migrator` in the same way, it just doesn't do anything.